Formula/c/conda-lock.rb

```diff
@@ -19,26 +19,13 @@ class CondaLock < Formula
 
   depends_on "rust" => :build # for pydantic
   depends_on "certifi"
+  depends_on "cryptography"
   depends_on "libyaml"
   depends_on "python@3.12"
 
-  on_linux do
-    depends_on "cryptography"
-
-    resource "jeepney" do
-      url "https://files.pythonhosted.org/packages/d6/f4/154cf374c2daf2020e05c3c6a03c91348d59b23c5366e968feb198306fdf/jeepney-0.8.0.tar.gz"
-      sha256 "5efe48d255973902f6badc3ce55e2aa6c5c3b3bc642059ef3a91247bcfcc5806"
-    end
-
-    resource "secretstorage" do
-      url "https://files.pythonhosted.org/packages/53/a4/f48c9d79cb507ed1373477dbceaba7401fd8a23af63b837fa61f1dcd3691/SecretStorage-3.3.3.tar.gz"
-      sha256 "2403533ef369eca6d2ba81718576c5e0f564d5cca1b58f73a8b23e7d4eeebd77"
-    end
```

Comment:
Just curious what changed in terms of the deps for jeepney/secretsstorage to be moved out of `on_linux` (so for all platforms)?

Reply:
nothing changed, its just a limitation of `update-python-resources` not being able to update these platform-specific resource blocks since it uses a naive regex. Seems like we have been forcing these deps on all platforms instead for convienence by using `extra_packages`: https://github.com/Homebrew/homebrew-core/blob/e97cf59c2f94a54d94e966b5151cd114e7c69e73/pypi_formula_mappings.json#L192